lib/jekyll/renderer.rb

```diff
@@ -67,6 +67,10 @@ def run
       output = convert(output)
       document.content = output
 
+      # Hook to modify converted content before layout is applied
+      document.trigger_hooks(:post_convert, output)
+      output = document.content
```

Comment:
I don't think this will do what you think it will do.

Can you write a test for this? `features/hooks.feature` is fine.